packages/ui/src/App.tsx

```diff
@@ -154,7 +154,7 @@ const Entry = React.memo(
                                     size="small"
                                     key={key}
                                     style={{marginRight: 10, ...valueStyles[key]}}
-                                    label={key + ': ' + value}
+                                    label={key + '=' + value}
```

Comment:
The chips looked better before, but it clearly makes sense if chips and queries use the same syntax.